eppo_client/rules.py

```diff
@@ -4,7 +4,8 @@
 from enum import Enum
 from typing import Any, List
 
-from eppo_client.base_model import SdkBaseModel
+from eppo_client.models import SdkBaseModel
+from eppo_client.types import AttributeType, ConditionValueType, SubjectAttributes
 
 
 class OperatorType(Enum):
```

Comment:
Would you be open to adding "IS_NULL", "IS_NOT_NULL" here or would you prefer that to be a minor version bump once we support it in the frontend?

Reply:
I think we should add it, and we can do that before publishing this version of the SDK, but let's move it to a different PR